### PR TITLE
Adjust autoscaller for dbs

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

File: kubernetes/cmsweb/hpa/dbs-hpa.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: dbs-global-r # should match deployment target
-  minReplicas: 8
-  maxReplicas: 10
+  minReplicas: 5
```
Comment (vkuznet) [inline kubernetes/cmsweb/hpa/dbs-hpa.yaml:11]: @muhammadimranfarooqi , @yuyiguo , I thought for DBS global-r we need range 8-10, and not 5-8. Please double check before I need to merge this PR.

@yuyiguo , please confirm that these changes reflect what is necessary for DBS.

File: kubernetes/cmsweb/hpa/dbs-hpa.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ spec:
     kind: Deployment
     name: dbs-phys03-r # should match deployment target
   minReplicas: 5
-  maxReplicas: 8
+  maxReplicas: 10
```
Comment (vkuznet) [inline kubernetes/cmsweb/hpa/dbs-hpa.yaml:62]: @muhammadimranfarooqi , I think Yuyi said that only global-r needs 8-10 range, the phys03-r/w instances can use 5-8. Again, please double-check with her and confirm.

File: kubernetes/cmsweb/hpa/dbs-hpa.yaml (hunk 1 above)
Comment (yuyiguo) [reply kubernetes/cmsweb/hpa/dbs-hpa.yaml:11]: @vkuznet @muhammadimranfarooqi 

It should be 8 to 10.  

Are you on vacation, Valentin? :-)

File: kubernetes/cmsweb/hpa/dbs-hpa.yaml (hunk 2 above)
Comment (yuyiguo) [reply kubernetes/cmsweb/hpa/dbs-hpa.yaml:62]: global-r/w needs 8-10 range,
phys03-r/w  5-8  range.

Comment (muhammadimranfarooqi) [reply kubernetes/cmsweb/hpa/dbs-hpa.yaml]: @yuyiguo @vkuznet 

I have adjusted replicas.

**Pull request merged**